scripts/docker/dockerfile_clang-6

```diff
@@ -0,0 +1,28 @@
+FROM axom/compilers:clang-6
+RUN sudo apt-get update -y
+RUN sudo apt-get install curl -fy
+
+RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
+ 
+#(For Clang Builds - Need to restore stdc++ lib)
+RUN sudo apt-get install libstdc++-7-dev
+
+RUN git clone --recursive https://github.com/LLNL/axom.git
+
+RUN git clone --recursive https://github.com/LLNL/RAJA.git
```

Comment:
Is the RAJA clone and build step needed? Doesn't the call to `uberenv` below (line 24) install RAJA? (Ditto for all the other docker files.)

Reply:
Keith had trouble getting this to work through docker.  

The plan is to fix this is a separate PR, after this one has merged.